Description: fix crash in python bindings
 The python bindings were using an incorrect search string to locate the DSO and
 would crash unless the -dev binary package was also installed.
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/evemu/+bug/1152774
Author: Stephen M. Webb <stephen.webb@canonical.com>

--- evemu-2.6.0.orig/python/evemu/base.py
+++ evemu-2.6.0/python/evemu/base.py
@@ -141,11 +141,11 @@ class LibC(LibraryWrapper):
     Wrapper for API calls to the C library.
     """
 
     @staticmethod
     def _cdll():
-        return ctypes.CDLL("libc.so.6", use_errno=True)
+        return ctypes.CDLL(None, use_errno=True)
 
     _api_prototypes = {
         "fdopen": {
             "argtypes": (c_int, c_char_p),
             "restype": c_void_p,
@@ -167,11 +167,11 @@ class LibEvdev(LibraryWrapper):
     Wrapper for API calls to libevdev
     """
 
     @staticmethod
     def _cdll():
-        return ctypes.CDLL("libevdev.so", use_errno=True)
+        return ctypes.CDLL("libevdev.so.2", use_errno=True)
 
     _api_prototypes = {
         #const char *libevdev_event_type_get_name(unsigned int type);
         "libevdev_event_type_get_name": {
             "argtypes": (c_uint,),
@@ -207,11 +207,11 @@ class LibEvdev(LibraryWrapper):
 class LibEvemu(LibraryWrapper):
     """
     Wrapper for API calls to the evemu library.
     """
 
-    _LIBNAME = "libevemu.so"
+    _LIBNAME = "libevemu.so.3"
 
     @staticmethod
     def _cdll():
         return ctypes.CDLL(LibEvemu._LIBNAME, use_errno=True)
 
